resources/mq_brokers.go

```diff
@@ -180,7 +180,6 @@ func MqBrokers() *schema.Table {
 			{
 				Name:        "aws_mq_broker_configurations",
 				Resolver:    fetchMqBrokerConfigurations,
-				Multiplex:   client.AccountRegionMultiplex,
```

Comment:
why did you remove the multiplex ?

Reply:
Apparently, it is a multiplex on a dependent table, which doesn't make sense (and it even prints a warning, that is why I noticed and removed it).

Reply:
Good catch